lib/extensions/build_context_extension.dart

```diff
@@ -39,6 +39,10 @@ extension ThemeModeExtension on BuildContext {
 
   Color get textHintColor =>
       isDarkMode ? AppColors.grayCool25 : AppColors.grayCool400;
+  Color get whiteColor => isDarkMode ? AppColors.white : AppColors.white;
+  Color get blackColor => isDarkMode ? AppColors.black : AppColors.black;
```

Comment:
colors here can't just be named `white color` or  `black color`, on what component this colour used? also check similar component that exist already and see what color was used on those